css/index.css

```
@@ -145,7 +145,10 @@
        line-height: 40px;
        font-size: 30px;
           } 
-About.active, project.active, experience.active, connect.active {
+    About.active, 
+    project.active, 
+    experience.active, 
+    connect.active {
```

If you want this to apply to everything that's active, an alternative is
```suggestion
.active {
```